src/components/GeriLifeTopbar.vue

```diff
@@ -1,66 +1,35 @@
 <template>
-  <q-toolbar class="bg-grey-4">
-    <div class="row" style="width: 100%">
-      <q-toolbar-title
-        @click="$route.path !== '/' && $router.push({ path: '/' })"
-        class="cursor-pointer col-md-2"
-      >
-        <q-img
-          width="30px"
-          height="30px"
-          src="~assets/gerilife-logo.png"
-          alt=""
-        />
-        <span class="text-grey-7"> GeriLife<sup>®</sup></span>
-      </q-toolbar-title>
-      <div v-if="!!getCookie('token')" class="col-12 col-md-5">
-        <q-btn
-          @click="$router.push({ path: '/residents' })"
-          class="text-grey-7"
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          flat
-          icon="fa fa-users"
-        >
-          <span class="q-ml-sm">{{
-            $i18n.t("mainLayoutNavbar-residentsLink")
-          }}</span>
-        </q-btn>
-        <q-btn
-          @click="$router.push({ path: '/activities' })"
-          class="text-grey-7"
-          flat
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          icon="fa fa-heartbeat"
-        >
-          <span class="q-ml-sm">{{
-            $i18n.t("mainLayoutNavbar-activitesLink")
-          }}</span>
-        </q-btn>
-        <q-btn
-          @click="$router.push({ path: '/homes' })"
-          class="text-grey-7"
-          flat
-          :size="isSmOrAbove ? 'md' : 'xs'"
-          icon="fa fa-home"
+  <div>
+    <q-toolbar class="bg-grey-4">
+      <div class="row no-wrap" style="width: 100%">
+        <q-toolbar-title
+          @click="$route.path !== '/' && $router.push({ path: '/' })"
```

Comment:
I'd prefer to not use this short-circuit approach. What happens when the router pushes the same route? What side effect are we mitigating?

Reply:
Relatedly, let's use the declarative `router-link` approach, rather than imperative `router.push`:

https://router.vuejs.org/api/#router-link